lucille-core/src/main/java/com/kmwllc/lucille/core/Runner.java

```diff
@@ -80,20 +80,21 @@ private Runner() {
    * no args: pipelines workers and indexers will be executed in separate threads within the same JVM; communication
    * between components will take place in memory and Kafka will not be used
    * <p>
-   * -usekafka: connectors will be run, sending documents and receiving events via Kafka. Pipeline workers
+   * -useKafka: connectors will be run, sending documents and receiving events via Kafka. Pipeline workers
    * and indexers will not be run. The assumption is that these have been deployed as separate processes.
    * <p>
-   * -local: modifies -usekafka so that workers and indexers are started as separate threads within the same JVM;
+   * -local: modifies -useKafka so that workers and indexers are started as separate threads within the same JVM;
    * kafka is still used for communication between them.
    * <p>
    * -render: prints out the effective/actual config in the exact form it will be seen by Lucille during the run
    */
   public static void main(String[] args) throws Exception {
     Options cliOptions = new Options()
-        .addOption(Option.builder("usekafka").hasArg(false)
-            .desc("Use Kafka for inter-component communication and don't execute pipelines locally").build())
+        .addOption(Option.builder("usekafka").hasArg(false).build())
+        .addOption(Option.builder("useKafka").hasArg(false)
+            .desc("Use Kafka for inter-component communication and don't execute pipelines locally. NOTE: This param is not case sensitive.").build())
```

Comment:
I think the param is _mostly_ case-sensitive -- it's only the k that's not... :) so, could just remove the NOTE

Reply:
I think we could get case-insensitivity for all params by applying toLowercase() to all the args before we pass them to the parser here       `cli = new DefaultParser().parse(cliOptions, args);`